php/composer.json

```diff
@@ -0,0 +1,19 @@
+{
+    "minimum-stability": "dev",
+    "repositories": [
+        {
+            "type": "path",
+            "url": "/sdk"
+        }
+    ],
+    "require": {
+        "hubspot/api-client": "*",
```

Comment:
What about locking api-client version?

Reply:
I've just deleted composer
There is only code base